slow_query.go

```diff
@@ -11,18 +11,38 @@ import (
 )
 
 var (
-	RegexSqlString = regexp.MustCompile("'[^']+'")
-	RegexNString   = regexp.MustCompile(`([0-9])+`)
-	RegexHasShard  = regexp.MustCompile(`(?i)from\s+(\w+|"\w+")\.`)
+	RegexSqlString  = regexp.MustCompile("('[^']+')") //Everything inside strings
+	RegexHasShard   = regexp.MustCompile(`(?i)from\s+(\w+|"\w+")\.`)
+	RegexIsDateTime = regexp.MustCompile(`(\d{4}-\d\d-\d\d \d\d:\d\d:\d\d.\d{6})`)
```

Comment:
We should probably wrap these with: `^'(stuff)'$` to assert that there is no offset match. Must be start of string, single quote, the regex matcher, single quote, end of string.

Otherwise stuff like `my secret is exposed because of the last little USR-123...` might make it through the filter (hits the unbounded guid matcher).

Reply:
ah you are right, I did that with the t/f already otherwise it would match on any string that had those letters. Will add.